contracts/src/TokenSpoke/TokenSpoke.sol

```diff
@@ -102,21 +102,31 @@ abstract contract TokenSpoke is ITokenSpoke, TeleporterOwnerUpgradeable, SendRee
     bool public isRegistered;
 
     /**
-     * @notice Fixed gas cost for performing a multi-hop transfer on the token hub contract,
-     * before forwarding to the destination token spoke instance.
+     * @notice Fixed gas cost for executing a multi-hop send message on the token hub contract,
+     * before forwarding to the destination token spoke instance. Note that for certain hub implementations,
+     * a higher required gas limit may be needed depending on the gas expenditure of the external call to the
+     * token contract to approve the spending of the optional secondary fee amount.
      */
-    uint256 public constant MULTI_HOP_REQUIRED_GAS = 240_000;
+    uint256 public constant MULTI_HOP_SEND_REQUIRED_GAS = 340_000;
+
+    /**
+     * @notice Fixed gas cost for executing a multi-hop call message on the token hub contract,
+     * before forwarding to the destination token spoke instance. Note that for certain hub implementations,
+     * a higher required gas limit may be needed depending on the gas expenditure of the external call to the
+     * token contract to approve the spending of the optional secondary fee amount.
+     */
+    uint256 public constant MULTI_HOP_CALL_REQUIRED_GAS = 350_000;
```

Comment:
is this relative to `MULTIHIOP_SEND_REQUEST_GAS`? Should we just do `MULTI_HOP_SEND_REQUIRED_GAS + 10_000`

Reply:
`_routeMultiHop` and `_routeMultiHopSendAndCall` are two separate (albeit similar) code paths in the `TokenHub` implementation, so the two values are technically independent of each other IMO.